app/models/state_file_ny_intake.rb

```diff
@@ -153,6 +153,17 @@ def calculated_refund_or_owed_amount
     calculator.refund_or_owed_amount
   end
 
+  def refund_or_owe_taxes_type
+    amt = calculated_refund_or_owed_amount
+    if amt == 0
+      :none
+    elsif amt > 0
+      :refund
+    elsif amt < 0
+      :owe
```

Comment:
Passing on these suggestions from rubocop, idk if it matters 

```suggestion
    if amt.zero?
      :none
    elsif amt.positive?
      :refund
    elsif amt.negative?
      :owe
```